src/app/_components/Header/Header.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
+'use client';
+
+import React, { useState } from 'react';
+import { Modal } from '../Modal';
+
+export const Header = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const toggleModal = () => {
+    setIsModalOpen(prevState => !prevState);
+  };
+
+  const Example = (
+    <div className="rounded-md bg-primary-300 p-4 shadow-md">
+      <h2>This is a Modal</h2>
+      <p>Modal content goes here.</p>
+    </div>
+  );
 
-export default function Header() {
   return (
-    <header className="bg-primary-100 p-5 text-h1 text-primary-700">
-      Header
+    <header className="flex justify-between bg-primary-100">
+      <h1 className="p-3 text-h1 text-primary-700"> Header </h1>
+      <button
+        onClick={toggleModal}
+        className="rounded-lg text-primary-500 outline"
+      >
+        Зворотній зв`язок
+      </button>
+
+      <Modal
+        isOpen={isModalOpen}
+        onClose={toggleModal}
+        isBlurBackground={false}
+      >
+        {Example}
+      </Modal>
```

Comment:
🟠 i think there is no need to put example modal into commits and codebase, you can add it to PR description ( and deploy live example from local to see it in action )